### PR TITLE
2.4.3 release/criticality-score/Portus

### PR DESCRIPTION
**Pull request by swetatech24**
https://github.com/Be-Secure/Be-Secure/issues/158
updated criticality score for Portus

### CODE REVIEW

File: grav/2.4.3/sast/Portus-2.4.3-codeql-report.json [hunk 1]
```diff
@@ -0,0 +1,740 @@
+ % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
```
Comment (sudhirverma) [inline grav/2.4.3/sast/Portus-2.4.3-codeql-report.json:1]: @swetatech24 can you remove this

**Review by sudhirverma — CHANGES_REQUESTED** [review]

File: grav/2.4.3/sast/Portus-2.4.3-codeql-report.json (hunk 1 above)
Comment (swetatech24) [reply grav/2.4.3/sast/Portus-2.4.3-codeql-report.json]: Removed sudhir

**Review by sudhirverma — APPROVED** [review]

File: grav/2.4.3/sast/Portus-2.4.3-codeql-report.json [hunk 2]
```diff
@@ -0,0 +1,738 @@
+[
```
Comment (sudhirverma) [inline grav/2.4.3/sast/Portus-2.4.3-codeql-report.json:1]: @swetatech24 you have add codeQL in grav project

**Review by sudhirverma — CHANGES_REQUESTED** [review]

File: grav/2.4.3/sast/Portus-2.4.3-codeql-report.json (hunk 2 above)
Comment (swetatech24) [reply grav/2.4.3/sast/Portus-2.4.3-codeql-report.json:1]: Ok I will delete it

**Pull request closed without merging**